adoc/chapters/programming_interface.adoc

```diff
@@ -21218,7 +21218,7 @@ In SYCL the half precision math functions are defined in
 [code]#sycl::half_precision#. The functions that are
 available within this namespace are specified in
 <<table.half.math.functions>>. These functions are
-implemented with a minimum of 10-bits of accuracy i.e. an ULP value is
+implemented with a minimum of 10-bits of accuracy i.e. the maximum error is
```

Comment:
Curious sentence indeed!
I wonder where it does come from.
Assuming we stick on this 10 bits with rounding to nearest, I guess it would be `(1 << 10)/2` ulp, which is 512 ulp.

Reply:
The point, if I understand it correctly, is that they return `float`, not `half`. And half precision is something like (1 << 13) ulp of `float`.